front/components/assistant_builder/AssistantTemplateModal.tsx

```diff
@@ -60,12 +54,7 @@ export function AssistantTemplateModal({
       <Page variant="modal">
         <div className="flex flex-col gap-5 pb-6">
           <div className="flex max-h-32 max-w-lg flex-row gap-3">
-            <Avatar
```

Comment:
:+1: 
Thinking out loud: would it make sense to remove emoji/bgcolor as props of avatar, and always use urls throughout our code?

Reply:
There are still some places where it's being used through emoji and background color.